split-rum/manifest.json

```diff
@@ -0,0 +1,43 @@
+{
+  "manifest_version": "2.0.0",
+  "app_uuid": "750d9e38-2856-44fe-98d0-9fbbc617d876",
+  "app_id": "split-rum",
+  "display_on_public_website": true,
+  "tile": {
```

Comment:
Could you add some uninstallation instructions in the readme and add a section here linking to them?